.dockerignore

```diff
@@ -0,0 +1,3 @@
+data/**
+models/**
+.git/**

```

Comment:
I'm actually not sure in practice... My understanding is that `data/**` explicitly ignores subdirectories, but it seems like just using `data` behaves the same way. It might just be an implicit vs. explicit difference.